app/src/main/java/org/sil/storyproducer/controller/accuracycheck/AccuracyCheckFrag.kt

```diff
@@ -83,7 +83,13 @@ class AccuracyCheckFrag : SlidePhaseFrag() {
                 button.background = greenCheckmark
                 Workspace.activeStory.slides[slideNum].isChecked = true
                 if (checkAllMarked()) {
-                    showConsultantPasswordDialog()
+                    // SP645 - BW 06/15/2022 new Affirm Accuracy Check dialog for SIL Story Producer

```

Comment:
This isn't going to work for optionally configuring a feature.  You should use Feature Flags (research possible libraries)

Reply:
one library Chris found for Feature Flags
https://medium.com/creditas-tech/feature-flags-using-kotlin-spring-boot-3786b78675d